packages/frontend/src/components/AddAppConnection/index.tsx

```diff
@@ -103,6 +104,17 @@ export default function AddAppConnection(
     [connectionId, key, steps, onClose],
   )
 
+  if (auth?.connectionType !== 'user-added') {
+    return (
+      <Dialog open={true} onClose={onClose}>
+        <DialogTitle>Error</DialogTitle>
+        <DialogContent>
+          <Infobox variant="error">Connections are not supported</Infobox>
+        </DialogContent>
+      </Dialog>
+    )
+  }
+
```

Comment:
nit: is there a modal in chakra to be used or migrate MUI in another PR?

Reply:
Yes good catch. I will migrate in a later PR. Please hold me accountable for this.